frontend/src/components/JobCard/JobCard.tsx

```diff
@@ -57,7 +58,7 @@ const JobCard: React.FC<JobCardProps> = ({ job }) => {
 
       <div className='card-footer'>
         <Typography variant='h4' component='h2'>
-          <BsPeople /> 00
+          <BsPeople /> {job.applicant_count}
```

Comment:
you declared that type can be also `null` `applicant_count: number | null;` if it is the case you should check if `applicant_count` is valid first before render. e.g `const applicantCount = job?.applicant_count ?? 0`

Reply:
ok..